tools/snp.sh

```diff
@@ -923,24 +926,101 @@ setup_guest_attestation() {
   echo "true" > "${guest_setup_file}"
 }
 
-get_cpu_code_name() {
-  local cpu_model=$(cat /proc/cpuinfo | grep "model name" | head -1 | cut -d ' ' -f5)
-  local cpu_code_name="milan"
+# Pass a function and a register to collect its value
+get_cpuid() {
```

Comment:
Variables in braces, keep spacing indentation consistent for this function and the other functions.